.pre-commit-config.yaml

```diff
@@ -57,3 +57,10 @@ repos:
     rev: v2.11
     hooks:
       - id: vulture
+  - repo: local
+    hooks:
+      - id: generate-manpages
+        name: Generate manpages
+        entry: podman run --rm ($ podman build -q -f Containerfiles/manpage.Containerfile)
+        language: system
+        files: 'convert2rhel/__init__.py'
```

Comment:
Actually, this might be difficult to do. I'm not sure but I couldn't get this to work locally. Maybe we just stick to the GitHub workflow. But for that to work we need to ensure that `podman run --rm $(podman build -q -f Containerfiles/manpage.Containerfile)` works locally